formatting/action.yml

```diff
@@ -20,7 +28,7 @@ runs:
         working-directory: ${{ inputs.path }}
         run: |
           set +e
-          grep --exclude="README.md" -rnI -e "[[:blank:]]$" .
+          grep --exclude={README.md,${{ inputs.exclude-files }}} --exclude-dir={${{ inputs.exclude-dirs }},''} -rnI -e "[[:blank:]]$" .
```

Comment:
Does this work if `inputs.exclude-files` or `inputs.exclude-dirs` are empty (the default value)?

Reply:
Yes, the command runs with the default value of '' for both `exclude-files` and `exclude-dirs`